ccl/backupccl: TestFullClusterBackup failed

ccl/backupccl.TestFullClusterBackup [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9339561?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/9339561?buildTab=artifacts#/) on release-23.1 @ [3f2adec1b34a86de342fea2811da936d1ae72a43](https://github.com/cockroachdb/cockroach/commits/3f2adec1b34a86de342fea2811da936d1ae72a43):


```
=== RUN   TestFullClusterBackup
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/47447a7ed84475b6aaa4b9399a882ce0/logTestFullClusterBackup494608081
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestFullClusterBackup
    testutils.go:199: no Invalid Descriptors
    testutils.go:199: no Invalid Descriptors
    full_cluster_backup_restore_test.go:350: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/47447a7ed84475b6aaa4b9399a882ce0/logTestFullClusterBackup494608081
--- FAIL: TestFullClusterBackup (253.48s)
=== RUN   TestFullClusterBackup/ensure_system_table_data_restored
    full_cluster_backup_restore_test.go:296: query 'SELECT * FROM system.scheduled_jobs': expected:
        852319971431219201, sql-schema-telemetry, 2023-03-30 12:03:33.142521 +0000 UTC, node, 2023-04-06 12:06:00 +0000 UTC, , 6 12 * * 4, ��, scheduled-schema-telemetry-executor, 
        I
        Gtype.googleapis.com/cockroach.sql.ScheduledSchemaTelemetryExecutionArgs
        852319971448324097, sql-stats-compaction, 2023-03-30 12:03:33.157522 +0000 UTC, node, NULL, 
        �pending, @hourly, ��, scheduled-sql-stats-compaction-executor, 
        K
        Itype.googleapis.com/cockroach.sql.ScheduledSQLStatsCompactorExecutionArgs
        852320675762765825, BACKUP 1680178028, 2023-03-30 12:07:08.070458 +0000 UTC, root, NULL, NULL, @hourly, , scheduled-backup-executor, 
        ��
        Htype.googleapis.com/cockroach.ccl.backupccl.ScheduledBackupExecutionArgs�F�DBACKUP TABLE data.public.bank INTO 'nodelocal://0/foo' WITH detached
        
        got:
        852319971431219201, sql-schema-telemetry, 2023-03-30 12:03:33.142521 +0000 UTC, node, 2023-03-30 12:06:00 +0000 UTC, 
        �pending, 6 12 * * 4, ��, scheduled-schema-telemetry-executor, 
        I
        Gtype.googleapis.com/cockroach.sql.ScheduledSchemaTelemetryExecutionArgs
        852319971448324097, sql-stats-compaction, 2023-03-30 12:03:33.157522 +0000 UTC, node, NULL, 
        �pending, @hourly, ��, scheduled-sql-stats-compaction-executor, 
        K
        Itype.googleapis.com/cockroach.sql.ScheduledSQLStatsCompactorExecutionArgs
        852320675762765825, BACKUP 1680178028, 2023-03-30 12:07:08.070458 +0000 UTC, root, NULL, NULL, @hourly, , scheduled-backup-executor, 
        ��
        Htype.googleapis.com/cockroach.ccl.backupccl.ScheduledBackupExecutionArgs�F�DBACKUP TABLE data.public.bank INTO 'nodelocal://0/foo' WITH detached
        
    --- FAIL: TestFullClusterBackup/ensure_system_table_data_restored (0.05s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73759 ccl/backupccl: TestFullClusterBackup failed [C-test-failure O-robot T-disaster-recovery branch-release-21.2]
</p>
</details>
/cc @cockroachdb/disaster-recovery
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestFullClusterBackup.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-26245

--- a/pkg/ccl/backupccl/full_cluster_backup_restore_test.go
+++ b/pkg/ccl/backupccl/full_cluster_backup_restore_test.go
@@ -286,6 +286,9 @@ CREATE TABLE data2.foo (a int);
 			case systemschema.CommentsTable.GetName():
 				query := fmt.Sprintf("SELECT comment FROM system.%s", table)
 				verificationQueries[i] = query
+			case systemschema.ScheduledJobsTable.GetName():
+				query := fmt.Sprintf("SELECT schedule_id, schedule_name FROM system.%s", table)
+				verificationQueries[i] = query
 			default:
 				query := fmt.Sprintf("SELECT * FROM system.%s", table)
 				verificationQueries[i] = query

